gtd-timer/Attributes/UserExistExceptionFilterAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System;
+using System.Net;
+
+using gtdtimer.Exceptions;
+
+namespace gtdtimer.Attributes
+{
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class)]
+    public class UserExistExceptionFilterAttribute : ExceptionFilterAttribute
+    {
+        public override void OnException(ExceptionContext context)
+        {
+            if (context.Exception is UserExistException)
+            {
+                context.HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
+                context.Result = new ContentResult { Content = context.Exception.Message };
```

Comment:
as we discussed - exception handling middleware would be better place for this logic

Reply:
It will be change after implement exception handling

Reply:
Please create task for it then

Reply:
Issue #29 created